src/pages/item/index.js

```diff
@@ -359,7 +359,7 @@ function Item() {
 
     const itemFleaFee = fleaFee(currentItemData.basePrice, currentItemData.lastLowPrice, 1, meta?.flea?.sellOfferFeeRate, meta?.flea?.sellRequirementFeeRate);
 
-    const traderIsBest = currentItemData.traderPriceRUB > currentItemData.lastLowPrice - itemFleaFee;
+    const traderIsBest = currentItemData.traderTotalPriceRUB > currentItemData.lastLowPrice - itemFleaFee;
```

Comment:
I see `traderTotalPriceRUB` valorized in the do fetch items only for `guns` and `presets`. How does this work for items that are not `guns`?

Reply:
Good catch!